core/domain/exp_domain.py

```diff
@@ -2427,9 +2427,9 @@ def update_states_from_model(
 
         Args:
             versioned_exploration_states: dict. A dict with two keys:
-                - states_schema_version: int. The states schema version for the
-                    exploration.
-                - states: dict. The dict of states comprising the exploration.
+                - states_schema_version: (int). The states schema version for
```

Comment:
Ditto, please keep this as it was, here and for "states" below.

Reply:
Done